svgdigitizer/__main__.py

```diff
@@ -122,6 +131,26 @@ def _create_svgplot(svg, sampling_interval, skewed):
     )
 
 
+def _create_citation(metadata):
+    r"""
+    Return a bibtex string built from a BIB file and a key provided in `metadata['source']['bib']`.
+
+    This is a helper method for :meth:`digitize_cv`.
+    """
+    from pybtex.database import parse_file
+
+    try:
+        metadata["source"]["bib"]
+    except KeyError as exc:
+        raise KeyError(
+            "The name of the bibfile must be specified in the metadata in `metadata['source']['bib']`."
+        ) from exc
+
+    bibfile = metadata["source"]["bib"]
+    bibliography = parse_file(f"{bibfile}.bib", bib_format="bibtex")
+    return bibliography.entries[bibfile].to_string("bibtex")
```

Comment:
This is intended for use with echemdb metadata. Of course, one could also think about providing a filename of the bib file as an argument to the flag. However, I guess automatisation will become complicated. It is possible that multiple files with different names, share a single BIB file.